packages/integration-tests/src/entities.ts

```diff
@@ -49,6 +49,8 @@ export enum Token {
   ETHER_PULL_Address = "0x0000000000000000000000000000000000008001",
   ETHER_Address = "0x0000000000000000000000000000000000000000",
   ETHER_ERC20_Address = "0x000000000000000000000000000000000000800A",
+  tokenTypeERC20 = "ERC20",
+  tokenTypeETH = "ETH",
```

Comment:
that is good that you have decided to move it separately, but then you have to align all tests:
![image](https://github.com/matter-labs/block-explorer/assets/51121511/2c95a12e-7cd2-4bf8-bc32-eb1c8cc137c8)
I would also suggest to simplify the name: 
```
ERC20_Name = "ERC20",
ETH_Name = "ETH",
```